CHANGELOG.md

```diff
@@ -1,15 +1,59 @@
 # PRQL Changelog
 
+## 0.3.2 — [unreleased]
+
+**Features**:
+
+- S-strings can [produce a full table.](https://prql-lang.org/book/language-features/s-strings.html#prql-3)
+- _Experimental:_ `switch` statement sets a variable to a value based on one of several expressions.
+  No page in the docs yet, but
+  [see this discussion](https://github.com/prql/prql/issues/1286#issue-1501645497)
+  for usage and the current syntax. _Note: this syntax may change._
+
+  ```
+  derive var = switch [
+  score <= 10 -> "low",
+  score <= 30 -> "medium",
+  score <= 70 -> "high",
+  true -> "very high",
+  ]
+  ```
+
+- _Experimental:_ `union` statement.
+  No page in the docs yet, but [see this PR](https://github.com/prql/prql/pull/894#issuecomment-1353548853) for usage.
+
+  ```
+  from employees
+  concat managers
+  union other_employees
+  ```
+
+- Any other `feat:` commits?
```

Comment:
Not on you to add them — do what you're happy doing — but some other ones we can add:
- Playground now live-queries some sample data https://github.com/prql/prql/commit/fadd91762705f791448a95e4be3798e90fb05234
- I add commits from newer contributors: https://github.com/prql/prql/commit/8f57f437dd05b161c9ff14e96b01f0f0addd620e
- Good docs improvement! https://github.com/prql/prql/commit/a90d5502b092e3369a38816de36b8782c9ea823a

Reply:
OK. Thanks!